src/cascadia/TerminalSettingsModel/DefaultTerminal.cpp

```diff
@@ -0,0 +1,91 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+#include "pch.h"
+#include "DefaultTerminal.h"
+#include "DefaultTerminal.g.cpp"
+
+#include <LibraryResources.h>
+
+using namespace winrt::Microsoft::Terminal::Settings;
+using namespace winrt::Microsoft::Terminal::Settings::Model::implementation;
+
+winrt::Windows::Foundation::Collections::IVector<Model::DefaultTerminal> DefaultTerminal::_available = winrt::single_threaded_vector<Model::DefaultTerminal>();
+std::optional<Model::DefaultTerminal> DefaultTerminal::_current;
+
+DefaultTerminal::DefaultTerminal(DelegationConfig::DelegationPackage pkg) :
+    _pkg(pkg)
+{
+}
+
+winrt::hstring DefaultTerminal::Name() const
+{
+    static const std::wstring def{ RS_(L"InboxWindowsConsoleName") };

```

Comment:
@DHowett when I made this one `std::wstring_view` and the `DefaultTerminal::Author` one below... static analysis flagged me as making a view to something that was going to disappear out of scope on the `RS_` macro.

Reply:
oh, it is a resource. my comment was about your non-resource impl. :smile: